test/parallel/test-worker-message-event.js

```diff
@@ -61,31 +61,31 @@ const dummyPort = new MessageChannel().port1;
   assert.throws(() => {
     new MessageEvent('message', { source: 1 });
   }, {
-    code: 'ERR_INVALID_ARG_TYPE',
-    message: /The "init\.source" property must be an instance of MessagePort/,
+    name: 'TypeError',
+    message: /Expected 1 to be an instance of MessagePort/,
   });
   assert.throws(() => {
     new MessageEvent('message', { source: {} });
   }, {
-    code: 'ERR_INVALID_ARG_TYPE',
-    message: /The "init\.source" property must be an instance of MessagePort/,
+    name: 'TypeError',
```

Comment:
Since we are changing error code this becomes a semver-major

Reply:
@KhafraDev it would be better if this PR wouldn't be semver-major, so we can land it in v22.